src/session/cookie-store/index.ts

```diff
@@ -31,20 +36,39 @@ export default class CookieSessionStore implements ISessionStore {
       return null;
     }
 
+    const { expiresAt, refreshToken } = unsealed as ISession;
+
+    // Check if the token has expired
+    if (refreshToken && expiresAt && expiresAt * 1000 < Date.now()) {
```

Comment:
Can we  play it a bit safer here? There is an edge case where we might have some clock skew where our code assumes the token is still valid but in reality it's not.

Could we for example add a skew of 1 minute? Where instead of checking `expiresAt` we check `expiresAt - 1 minute`?